server/bundles/io.cloudbeaver.model/src/io/cloudbeaver/model/app/WebAppConfiguration.java

```diff
@@ -41,4 +41,8 @@ public interface WebAppConfiguration {
     boolean isFeaturesEnabled(String[] requiredFeatures);
 
     boolean isFeatureEnabled(String id);
+
+    default boolean isPrivateProjectsAccessible() {
```

Comment:
```suggestion
    default boolean isSupportsCustomConnections() {
```
we can use already existing method